src/torcontrol.cpp

```diff
@@ -631,10 +634,14 @@ void TorController::disconnected_cb(TorControlConnection& _conn)
     if (!reconnect)
         return;
 
-    LogDebug(BCLog::TOR, "Not connected to Tor control port %s, trying to reconnect\n", m_tor_control_center);
-
     // Single-shot timer for reconnect. Use exponential backoff.
-    struct timeval time = MillisToTimeval(int64_t(reconnect_timeout * 1000.0));
+    int64_t timeout = std::min(reconnect_timeout, RECONNECT_TIMEOUT_MAX);
```

Comment:
`timeout` should be a float, this a comparison on two float values returning a float.